tests/conftest.py

```diff
@@ -63,6 +71,23 @@ def setup_test(test_dir: Path, macaron_path: Path) -> NoReturn:  # type: ignore
     defaults.clear()
 
 
+@pytest.fixture()
+def check_result() -> CheckResult:  # pylint: disable=unused-argument
+    """Create a CheckResult instance.
+
+    Parameters
+    ----------
+    setup_test
+        Depends on setup_test fixture.
+
+    Returns
+    -------
+    CheckResult
+        The CheckResult instance.
+    """
+    return CheckResult(justification=[])  # type: ignore
```

Comment:
It's better to instantiate `CheckResult` directly in tests to set the expected output. This fixture is not doing much and I'm not sure if it's helpful.

Reply:
Fixed in [29cdd3a](https://github.com/oracle-samples/macaron/pull/67/commits/29cdd3a9e361a0f5f29c835805a2d9f186b92d0a).